From 8efcd45dc2c6b68351cfd4894828c016995917f3 Mon Sep 17 00:00:00 2001
Message-Id: <8efcd45dc2c6b68351cfd4894828c016995917f3.1241177871.git.joerg@alea.gnuu.de>
In-Reply-To: <e879f68a77182a30e0ee2c1ffaea9ae9cd12f326.1241177871.git.joerg@alea.gnuu.de>
References: <e879f68a77182a30e0ee2c1ffaea9ae9cd12f326.1241177871.git.joerg@alea.gnuu.de>
From: =?utf-8?q?J=C3=B6rg=20Sommer?= <joerg@alea.gnuu.de>
Date: Fri, 1 May 2009 13:31:25 +0200
Subject: [PATCH 4/4] isdeletedFile: Remove useless check for deleted files

Above the check for /tmp/mutt is already a check for /tmp, which already
catches all those files of mutt.
---
 checkrestart |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/checkrestart b/checkrestart
index 20f2121..34e811c 100755
--- a/checkrestart
+++ b/checkrestart
@@ -280,9 +280,6 @@ def isdeletedFile (f):
     # Skip automatically generated files
     if f.endswith('icon-theme.cache'):
         return 0
-    # Skip mutt files
-    if re.compile("/tmp/mutt-\w+-[\-0-9]").search(f):
-        return 0
     # Skip font files
     if f.startswith('/var/cache/fontconfig/'):
         return 0
-- 
1.6.3.rc1

